Runtime/Scripts/BaseMeshSync.Textures.cs

```diff
@@ -75,6 +78,23 @@ partial class BaseMeshSync {
         /// <param name="mat">Material to set the texture on</param>
         /// <param name="textureName">Name of the texture in the material</param>
         /// <returns></returns>
+        private bool SetSerializedTextureForMaterialOrTryAgainLater(Material mat, string textureName) {
+            if (!SetSerializedTextureForMaterial(mat, textureName)) {
+                EditorApplication.delayCall -= UpdatePendingMaterials;
+                EditorApplication.delayCall += UpdatePendingMaterials;
+
+                return false;
+            }
+
+            return true;
+        }
```

Comment:
You would have to move the try catch block here. Otherwise if there is Unity Exception, it will register again, which is not the behavior in the previous version